### PR TITLE
LPS-148844 Include new businessType (Rich Text)

### PR DESCRIPTION
**Pull request by MarinhoFeliphe**
Story: [LPS-143064](https://issues.liferay.com/browse/LPS-143064)

### CODE REVIEW

**Review by gabrielwas — CHANGES_REQUESTED** [review]

**Comment by gabrielwas** [comment]
@MarinhoFeliphe sorry for the delayed review :/ Can you check the comment and fix the conflicts? Thanks :D

File: modules/apps/object/object-service/src/main/java/com/liferay/object/model/impl/ObjectEntryImpl.java [hunk 1]
```diff
@@ -87,6 +90,14 @@ public String getTitleValue() throws PortalException {
 			if (objectField != null) {
 				Map<String, Serializable> values = getValues();
 
+				if (Objects.equals(
+						objectField.getBusinessType(),
+						ObjectFieldConstants.BUSINESS_TYPE_RICH_TEXT)) {
+
+					return HtmlUtil.extractText(
+						String.valueOf(values.get(objectField.getName())));
```
[the remark this answers is not in the document]
Comment (MarinhoFeliphe) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/model/impl/ObjectEntryImpl.java:98]: Hi @gabrielwas, are these the places that you mentioned?

[ObjectRecipientCommerceDefinitionTermContributor](https://github.com/liferay-objects/liferay-portal/blob/master/modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/order/term/contributor/ObjectRecipientCommerceDefinitionTermContributor.java#L111-L113)

[ObjectCommerceDefinitionTermContributor](https://github.com/liferay-objects/liferay-portal/blob/master/modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/order/term/contributor/ObjectCommerceDefinitionTermContributor.java#L101-L103)

[ObjectEntryLocalServiceImpl](https://github.com/liferay-objects/liferay-portal/blob/master/modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java#L1772-L1773)

[the remark this answers is not in the document]
Comment (gabrielwas) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/model/impl/ObjectEntryImpl.java:98]: Actually, all the places that we call `HtmlUtil.extractText` to get the objectField value in this PR. I was wondering if we can extract this logic (this if/else statement to check the businessType and then retrieving the value) to a util class or something. If you'd like, we can quick chat about it

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @MarinhoFeliphe, can you please check if these failures are related to your changes? Thanks

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @MarinhoFeliphe, can you please check if these failures are related to your changes? Thanks

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by MarinhoFeliphe** [comment]
ci:forward

**Pull request closed without merging**